### PR TITLE
rfc 612

### PR DESCRIPTION
**Pull request by coury-clark**
Adding the summary of changes in [RFC 612 (patch release and responsibility)](https://docs.google.com/document/d/1Gecnsk4mnmf_p9SO4ExICSTC_op-eL2I_kwutHvwSmU/edit#heading=h.trqab8y0kufp) to the handbook.

### CODE REVIEW

**Comment by coury-clark** [comment]
Patch request template update: https://github.com/sourcegraph/sourcegraph/pull/34711

File: content/departments/product-engineering/engineering/process/releases/index.md [hunk 1]
```diff
@@ -4,18 +4,23 @@ This document describes how we release Sourcegraph.
...
 
-> As of Feb 3, 2022, the role of Release Captain is now rotation-based within the [Cloud](../../cloud/index.md) org, [learn more](https://sourcegraph.slack.com/archives/C0EPTDE9L/p1643915838386829). If you need assistance with releases, please reach out to [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX). The [Delivery] team is more than happy to help you through the process.
+> As of [RFC 612](https://docs.google.com/document/d/1Gecnsk4mnmf_p9SO4ExICSTC_op-eL2I_kwutHvwSmU/edit) the ownership of the release process has changed.
+> The [Release Guild](./release_guild.md) is now the entity that owns the release process. Read more details about release
+> responsibilities in the [Releases](./index.md#Releases) section below.
 
 ### Releases
 
-**Sourcegraph releases are monthly.**
+**Sourcegraph minor releases are monthly.**
```
Comment (kevinwojo) [inline content/departments/product-engineering/engineering/process/releases/index.md:13]: Begs the question, when will we ever do a major release? And if so, who owns that?

**Review by kevinwojo — APPROVED** [review]
Just a philosophical question rather than any hard objections.

File: content/departments/product-engineering/engineering/process/releases/index.md (hunk 1 above)
Comment (unknwon) [reply content/departments/product-engineering/engineering/process/releases/index.md:13]: It's been years for a major release 😂

I think the proposed change in the title at least prevents from release guild automatically owning the major ones, and is technically correct (to me), which forces the leadership to think about it when we plan to make a major release.

File: content/departments/product-engineering/engineering/process/releases/index.md [hunk 2]
```diff
@@ -31,9 +36,36 @@ A _patch release_ refers to a patch version increase of Sourcegraph (e.g. `3.0.0
...
+      We require that the guild still be notified about any patch requests to be aware of any patches that are currently in progress.
+      > In most cases, waiting until the next [full release](#releases) is the best approach.
+
+#### Patch request process
+
+The release guild will evaluate this patch process at the end of Q2 FY23 with the goal to make the patch requestor
+the first and primary party responsible to cut the patch ([read more in RFC 612](https://docs.google.com/document/d/1Gecnsk4mnmf_p9SO4ExICSTC_op-eL2I_kwutHvwSmU/edit)). Some critiera that we will use to measure success
+will be:
+
+1. No significant disruption to the overall customer upgrade experience
```
Comment (unknwon) [inline content/departments/product-engineering/engineering/process/releases/index.md]:
```suggestion
1. Only include absolute necessary changes to address issues introduced in the corresponding minor release, and no disruption to the overall customer upgrade experience.
```

I'd like frame it the other way around, WDYT?

**Review by unknwon — APPROVED** [review]
Thank you!

**Pull request merged**